include/linker/ramfunc.ld

```diff
@@ -0,0 +1,17 @@
+SECTION_DATA_PROLOGUE(.ramfunc,,)
```

Comment:
@SebastianBoe could you review just the linker script file?

Reply:
LGTM